src/main/java/com/iuc/requests/controller/UserController.java

```diff
@@ -1,42 +1,99 @@
 package com.iuc.requests.controller;
 
 import com.iuc.requests.dto.StaffDto;
+import com.iuc.requests.dto.StudentDto;
 import com.iuc.requests.service.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+import org.modelmapper.ModelMapper;
 
 import javax.validation.constraints.Email;
+import javax.validation.constraints.Pattern;
 import java.util.List;
 
 @RestController
 @RequestMapping("/iuc/users")
 public class UserController {
 
-    @Autowired
-    private UserService userService;
+  @Autowired private UserService userService;
+  @Autowired ModelMapper modelMapper;
 
-    @GetMapping("/staff")
-    public StaffDto findStaffByEmail(@RequestParam(value = "email") @Email String email) {
-        return null;
-    }
+  @GetMapping("/staff")
+  public StaffDto findStaffByEmail(@RequestParam(value = "email") @Email String email) {
+    return modelMapper.map(userService.findStaffByEmail(email), StaffDto.class);
```

Comment:
``userService.findStaffByEmail(email)`` should return a ``StaffDto``.  Do ``modelMapper.map`` in the service.